app/routes/events/utils.js

```diff
@@ -1,36 +1,45 @@
 // @flow
 import { pick } from 'lodash';
 import moment from 'moment-timezone';
-import type { TransformEvent, EventType } from 'app/models';
+import type { TransformEvent } from 'app/models';
 
-export const eventTypes = {
-  company_presentation: 'Bedriftspresentasjon',
+// Value type based on the EVENT_CONSTANTS
+export type eventTypes = $Values<typeof EVENT_CONSTANTS>;
+
+// Current eventTypes
+export const EVENT_CONSTANTS: eventTypes = {
+  company_presentation: 'Bedpres',
```

Comment:
Not sure, but think i prefer `Bedriftspresentasjon`, but for `Alternativ Bedriftspresentasjon` may be a bit too long. Have you tested how that works (and maybe increase the size of the list. :thinking:

Reply:
I also prefer `Bedriftspresentasjon`, but as you say `Alternativ Bedriftspresentasjon` is quite long. I tested that and the text will wrap almost everywhere it's used.

Not sure what you mean by "_and maybe increase the size of the list_"? Like the number of elements or the width, to remove the wrapping?

Reply:
> and maybe increase the size of the list

0/10 phrasing 😅

> I tested that and the text will wrap almost everywhere it's used.


Yee, "Alternativ bedpres" is probably the way to go.